types/taiko/index.d.ts

```diff
@@ -171,6 +171,14 @@ export interface Response extends BasicResponse {
   redirectedResponse?: BasicResponse[];
 }
 
+export interface ForceOption {
+  force?: boolean;
+}
+
+export interface ForcedNavigationOptions extends NavigationOptions {
```

Comment:
Can we update it to 
`export interface ForcedNavigationOptions extends NavigationOptions, ForceOption {}`

Reply:
Smart idea, thanks! Will adjust accordingly and push soon.